torch/nn/modules/pooling.py

```diff
@@ -130,7 +131,10 @@ class MaxPool2d(_MaxPoolNd):
 
     Shape:
         - Input: :math:`(N, C, H_{in}, W_{in})`
-        - Output: :math:`(N, C, H_{out}, W_{out})`, where
+        - Output: :math:`(N, C, H_{out}, W_{out})` or
```

Comment:
For consistency with other modules, both possible shapes should be on a single Input and Output line (check out [nn.Upsample](https://pytorch.org/docs/stable/generated/torch.nn.Upsample.html) as an example).